package.json

```diff
@@ -8,7 +8,7 @@
     "analyze": "cross-env ANALYZE=true NODE_ENV=production DISABLE_FEATURES=wpcom-user-bootstrap CALYPSO_CLIENT=true webpack",
     "eslint": "eslint tasks/ client/",
     "eslint:fix": "eslint --fix tasks/ client/",
-    "test": "npm run test-client && npm run eslint",
```

Comment:
Should `npm run eslint` be added to the Husky `pre-commit`?

Reply:
Yep, this is already in `develop`. https://github.com/Automattic/woocommerce-services/blob/develop/lint-staged.config.js#L3